lm-agent/lm_agent/server_interfaces/flexlm.py

```diff
@@ -57,18 +57,15 @@ async def get_report_item(self, product_feature: str):
         parsed_output = self.parser(server_output)
 
         # raise exception if parser didn't output license information
-        if parsed_output.get("total") is None or any(
-            [
-                parsed_output.get("total", {}).get("used") is None,
-                parsed_output.get("total", {}).get("total") is None,
-            ]
+        if any(
+            [parsed_output is None, parsed_output.get("total") is None, parsed_output.get("used") is None]
```

Comment:
I _think_ you could demorgan this to get:

```suggestion
        if not all([parsed_output, parsed_output.get("total"), parsed_output.get("used")]):
```

Reply:
When `parsed_output.get("used")` is `0`, `not all([...])` fails because it evaluates to `False`.